ulid/test.ts

```diff
@@ -149,7 +149,7 @@ Deno.test("ulid", async (t) => {
   );
 });
 
-Deno.test("monotonicity", async (t) => {
+Deno.test("stubbedUlid() handles monotonicity", async (t) => {
```

Comment:
This indirectly tests `monotonicUlid` (`stubbedUlid` isn't a public API)

Reply:
Ah, yes. I'll fix it as I missed it.

Reply:
ref: https://github.com/denoland/deno_std/pull/4075
Thanks